### PR TITLE
Some edits to test_data.py

### PR DESCRIPTION
**Pull request by abotiamnot**
This will work with #395 

Will fix the issues added to that.

### CODE REVIEW

File: heliopy/data/test/test_data.py [hunk 1]
```diff
@@ -165,14 +165,15 @@ def test_mag320ms(self):
         check_datetime_index(df)
 
     def test_mag15s(self):
-        df = imp.mag15s(self.probe, self.starttime, self.endtime)
```
Comment (dstansby) [inline heliopy/data/test/test_data.py]: This shouldn't make a difference, since `self.probe` is `'8'` anyway I think.

Comment (abotiamnot) [reply heliopy/data/test/test_data.py]: Oh yep, so sorry, forgot to remove this!

**Comment by dstansby** [comment]
Don't think this fixes anything, but tidies things up a little bit 👍

**Comment by abotiamnot** [comment]
@dstansby Yep, none. Just tidies up somethings. I thought there was some problem with test_data.py!

**Pull request merged**